output/opendap_post2.sh

```diff
@@ -77,9 +89,29 @@ if [[ ${#SERVERS[@]} -eq 0 ]]; then
    fi
    exit
 fi
+
+#+ vvvvv
+# BEGIN - determine what files to upload, takes into
+# consideration files written in the scenario's run.properties
+# file, using 2 keys:
+#   + post.opendap.files           - generated by output/createOPeNDAPFileList.sh
+#   + post.opendap.additionalFiles - specified by user config via "postAdditionalFiles"
+#
+declare -a FILES_Standard
+fileList=${properties["post.opendap.files"]}
+IFS=' ' read -r -a FILES_Standard <<< "$fileList"
+
+declare -a FILES_Additional
+additionalFileList=${properties["post.opendap.additionalFiles"]}
```

Comment:
The `post.opendap.files` property only consists of files that are confirmed to exist (at least), because the script that creates the list checks with either `ls` or `-e` (in all cases?). But this file list has been formed without confirming that the files in the list both exist and are not zero length. Not sure what is the best way to handle this. Maybe add a check and emit a warning message if they are not found or are zero length? And then remove them from the list of files to be transferred?